### PR TITLE
Comic Extra - Update to new URL and page selector

### LINKED ISSUE #1541: Comicextra still notices updates but can't download pages / Latest not working
### Source information

Comicextra 1.4.14

### Source language

English

### Steps to reproduce

Add any series on Comicextra to library
It still loads chapter lists and notices when updates happen
Errors out when trying to download pages
Webview leads to the right website 

In source menu, popular tab still works, but latest and searching seems to be broken too.

Likely just some slight url changes?

### Expected behavior

Should download, populate latest tab, etc.

### Actual behavior

Endless loading or sad emoji screen.

### Mihon/Tachiyomi version

tachiyomi 1.10.4

### Android version

Android 11

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.15.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### LINKED ISSUE #1951: ComicExtra: URL redirects/changed
### Source information

ComicExtra 1.4.14

### Source language

English

### Source new URL

https://comicextra.org/

### Other details

Latest page doesn't load, Popular page does load but no pages found for chapters

### Acknowledgements

- [X] I have updated all installed extensions.
- [X] I have opened WebView and checked that the source URL has not changed yet.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [ ] I have admitted that I am a clown by having checked this box, as I have not read these acknowledgements.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by barrelltitor**
I have previously made another PR where I improperly tested this and as a result this was not properly fixed. This time I made sure to test on a clean emulator and also provide video to not waste anybody's time

https://github.com/keiyoushi/extensions-source/assets/108460132/9ba8814b-3788-4f1d-a064-948e0e4fd7dc

Closes #1951 
Closes #1541 

studio64_89FUlbl7qZ
studio64_TzhD6s0pHx

In case there are any issues please let me know and I will fix them right away!

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [x] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [x] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [x] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

**Comment by barrelltitor** [comment]
studio64_HhEUYtOAPo
Fixed search as well

File: src/en/comicextra/src/eu/kanade/tachiyomi/extension/en/comicextra/ComicExtra.kt [hunk 1]
```diff
@@ -40,7 +40,7 @@ class ComicExtra : ParsedHttpSource() {
 
     override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
         return if (query.isNotBlank()) {
-            GET("$baseUrl/comic-search?key=$query", headers)
+            GET("$baseUrl/search?keyword=$query", headers)
```
Comment (vetleledaal) [inline src/en/comicextra/src/eu/kanade/tachiyomi/extension/en/comicextra/ComicExtra.kt]: Use `addQueryParameter` so the query can contain special characters.
Should also have pagination

**Review by vetleledaal — CHANGES_REQUESTED** [review]

File: src/en/comicextra/src/eu/kanade/tachiyomi/extension/en/comicextra/ComicExtra.kt (hunk 1 above)
Comment (barrelltitor) [reply src/en/comicextra/src/eu/kanade/tachiyomi/extension/en/comicextra/ComicExtra.kt]: I have made the requested changes

Comment (vetleledaal) [reply src/en/comicextra/src/eu/kanade/tachiyomi/extension/en/comicextra/ComicExtra.kt]: CI failed, you also need the imports

Comment (barrelltitor) [reply src/en/comicextra/src/eu/kanade/tachiyomi/extension/en/comicextra/ComicExtra.kt]: Hmm it's still failing. One moment let me see what's up, I've tested the change before committing I must have removed something by mistake

**Review by vetleledaal — APPROVED** [review]

**Pull request merged**